sympy/functions/combinatorial/numbers.py

```diff
@@ -866,12 +872,8 @@ class harmonic(Function):
     >>> limit(harmonic(n, 3), n, oo)
     -polygamma(2, 1)/2
 
-    However we cannot compute the general relation yet:
-
-    >>> limit(harmonic(n, m), n, oo)
-    harmonic(oo, m)
-
-    which equals ``zeta(m)`` for ``m > 1``.
+    >>> limit(harmonic(n, m+1), n, oo) # does not hold for m == 1
+    zeta(m + 1)
```

Comment:
Why doesn't this hold for `m == 1`?
```python
In [16]: limit(harmonic(n, m+1), n, oo)
Out[16]: ζ(m + 1)

In [17]: limit(harmonic(n, m+1), n, oo).subs(m, 1)
Out[17]: 
 2
π 
──
6 

In [18]: limit(harmonic(n, m+1).subs(m, 1), n, oo)
Out[18]: 
 2
π 
──
6 

In [19]: harmonic(n, m+1).subs(m, 1).subs(n, 1e6)
Out[19]: harmonic(1000000.0, 2)

In [20]: harmonic(n, m+1).subs(m, 1).subs(n, 1e6).n()
Out[20]: 1.64493306684873

In [21]: harmonic(n, m+1).subs(m, 1).subs(n, 1e12).n()
Out[21]: 1.64493406684723

In [22]: N(pi**2/6)
Out[22]: 1.64493406684823
```

Reply:
I was about to clarify in the documentation about this but you merged too soon. The comment `does not hold for m == 1` refers to $\lim_{n\to\infty}H_{n,m}=\zeta(m)$ not holding if the _second argument of `harmonic()` is 1, not the second argument of the given expression `harmonic(m+1)`.

Reply:
We can update that in one of the other PRs then.

I see what you mean so the doctest uses `m+1` with `m` a positive integer because we want an integer greater than 1 for the second argument.

If the second argument is `1` then we get:
```python
In [9]: limit(harmonic(n, 1), n, oo)
Out[9]: ∞

In [10]: limit(harmonic(n, m+1), n, oo).subs(m, 0)
Out[10]: zoo
```